src/System.Net.Sockets/src/Resources/Strings.resx

```diff
@@ -244,4 +244,7 @@
   <data name="InvalidOperation_MultipleContinuations" xml:space="preserve">
     <value>Another continuation was already registered.</value>
   </data>
-</root>
+  <data name="net_sockets_sendpackelement_FileStreamMustBeAsync" xml:space="preserve">
+    <value>The FileStream must be opened asynchronously.</value>
```

Comment:
Nit: "The FileStream must have been opened for asynchronous reading and writing."